assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/updater/SW360UpdaterTest.java

```diff
@@ -166,7 +166,7 @@ private Path writeCsvFile() throws IOException {
         }
         String csvContent = String.format("Artifact Id,Group Id,Version,Coordinate Type,Clearing State,Clearing Document%s" +
                 "test,test,x.x.x,mvn,%s,%s%s", System.lineSeparator(), clearingState, clearingDocPath, System.lineSeparator());
-        FileUtils.writeStringToFile(tempCsvFile, csvContent, StandardCharsets.UTF_8);
+        Files.write(tempCsvFile.toPath(), csvContent.getBytes(StandardCharsets.UTF_8));
```

Comment:
`tempCsvFile` (and thus parent directory) exists due to the `newFile` in line 161.